### PR TITLE
Delete IsRuntimeImplemented

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.csproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="5.0.0" />
```
Comment (jkotas) [inline src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.csproj:22]: Undoing unnecessary diff

**Review by MichalStrehovsky — APPROVED** [review]
Thank you!
